convert2rhel/subscription.py

```diff
@@ -81,24 +86,27 @@ def register_system():
     """Register OS using subscription-manager."""
 
     # Loop the registration process until successful registration
-    while True:
+    attempt = 0
+    while True and attempt < MAX_NUM_OF_ATTEMPTS_TO_SUBSCRIBE:
         registration_cmd = get_registration_cmd()
-        loggerinst.info("Registering system by running subscription-manager"
-                        " command ... ")
+        loggerinst.info("Attempt %s of %s: Registering system by running subscription-manager"
+                        " command ... ", attempt+1, MAX_NUM_OF_ATTEMPTS_TO_SUBSCRIBE)
```

Comment:
I'd revert this and instead have a logger at the end just before sleep. Most of the time people will be successful in registering so displaying "Attempt 1 of 3..." here can be confusing

Reply:
good catch but since regression & verification is already in progress + given the time pressure I'd postpone this UX question for later issue/pr.